recipe/install_binutils.sh

```diff
@@ -7,10 +7,12 @@ cd build
 make install-strip
 export HOST="${ctng_cpu_arch}-conda-linux-gnu"
 export OLD_HOST="${ctng_cpu_arch}-${ctng_vendor}-linux-gnu"
+mkdir -p $PREFIX/$OLD_HOST/bin
 # Remove hardlinks and replace them by softlinks
 for tool in addr2line ar as c++filt dwp elfedit gprof ld ld.bfd ld.gold nm objcopy objdump ranlib readelf size strings strip; do
   rm -rf $PREFIX/$HOST/bin/$tool
```

Comment:
I just realized, if this is in the sysroots, then how does this delete propagate downstream without creating a conflict in the packages later?

Reply:
I don't understand. See the comment at line 11.

Reply:
Isn't `$PREFIX/$HOST/bin/$tool` from the sysroot-linux-64 package?

Reply:
No, these tools are built by `binutils`.